ui_framework/src/global_styling/variables/_typography.scss

```diff
@@ -32,7 +34,23 @@ $kuiFontSizeXXL:    48px;
 
 // Line height
 
-$kuiLineHeight: 1.5 * $kuiFontSize;
+$kuiLineHeight:     1.5 * $kuiFontSize;
+
+$kuiLineHeightXS:   $kuiLineHeight * 0.75;
+$kuiLineHeightS:    $kuiLineHeight * 0.75;
+$kuiLineHeightM:    $kuiLineHeight;
+$kuiLineHeightL:    $kuiLineHeight * 1.5;
+$kuiLineHeightXL:   $kuiLineHeight * 2;
```

Comment:
I'll need to tweak this stuff separately. Some of it's too fat. I'll do it in another PR.